plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -85,11 +86,11 @@
     choices: ["guest", "reporter", "developer", "master", "maintainer", "owner"]
   state:
     description:
-      - create or delete group.
-      - Possible values are present and absent.
+      - create, delete or block a user.
+      - Possible values are present, absent, blocked, and unblocked.
```

Comment:
```suggestion
      - Create, delete or block a user.
```
We have possible options in the ```choices``` field, so, it's extra information in the parameter description.

Reply:
Ah, yes, I see how the docs get updated with that. Indeed extra unnecessary information